File: src/main/java/guess/Gamefor.java
package guess;

import java.util.Random;
import java.util.Scanner;

public class Gamefor {
    public static void main(String[] args) {
        int secret = new Random().nextInt(10) + 1;
        System.out.println(secret);
        for(int i = 0;i <= 4;i++){
            Scanner scanner = new Scanner(System.in);
            System.out.print("Please enter a number("+i+"/4)");
            int number = scanner.nextInt();
            if(number < secret){
                System.out.println("higher");
            }else if(number > secret){
                System.out.println("low");
            }else{
                System.out.println("Great!, the number is " + number);
                break;
            }
        }
        System.out.println("結束");

    }

}

File: src/main/kotlin/com/tom/Gamefor.kt
package com.tom

import java.util.*

fun main() {
    var random =Random().nextInt(10)+1
    val scanner = Scanner(System.`in`)
    println(random)
    var number = 0
    for (i in 1..4){
        print("Please enter a number(${i}/4) : ")
        var number = scanner.nextInt()
        if(number > random)
            println("low")
            else if(number < random)
            println("high")
        else {
            println("Great!,the number is $number")
            break
        }

    }
    print("結束")
}